test_zfs_sr.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from lib.common import wait_for
+import time
+
+# Requirements:
+# - one XCP-ng host >= 8.2 with an additional unused disk for the SR
+# - access to XCP-ng RPM repository from the host
+
+@pytest.fixture(scope='module')
```

Comment:
Not needed anymore, a `sr_disk` fixture is now available by default (through `conftest.py`)